core/unix/ksynch_macos.c

```diff
@@ -41,6 +41,7 @@
  */
 
 #include "../globals.h"
+#include <errno.h>
```

Comment:
Hmm it may be safest to #undef errno after including this to ensure we have no refs to it

Reply:
Done.